README.md

```diff
@@ -31,7 +31,7 @@
 
 The Puppet dsc module manages Windows PowerShell DSC (Desired State Configuration) resources.
 
-This module generates Puppet types based on DSC resources' MOF (Managed Object Format) schema files.
+This module generates Puppet types based on DSC Resources MOF (Managed Object Format) schema files.
 
 In this version, the following DSC Resources are already built and ready for use:
 
```

Comment:
Actually no, we included WMF 5 built-in MOFs, not 5.1 or later so the sentence is still accurate.